### ISSUE TITLE
Cannot format code on Vs Code

### ISSUE THREAD

**Issue by iphyman** [mytonwallet/vscode-tonsolidity-extension#1]
![image](https://github.com/mytonwallet/vscode-tonsolidity-extension/assets/86075509/eaacb9cb-e9dd-4b8a-8351-8db94e9600af)

**Comment by mytonwallet** [comment]
Yes, we are working on it. In couple of weeks we will issue the updates by many things.
